__tests__/app.test.js

```diff
@@ -323,3 +324,31 @@ describe("DELETE /api/comments/:comment_id", () => {
     })
   })
 });
+
+describe('GET /api/users', ()=>{
+    test('200: responds with an array of users object', ()=>{
+        return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.users.length).toBe(4);
+        const {users} = body
+        users.forEach((user) => {
+          expect(user).toMatchObject({
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String)
+          });
+        });
+      });
+  });
+    test("404: responds with an error message for a not found path", () => {
+        return request(app)
+          .get("/api/notauser")
+          .expect(404)
+          .then(({ body }) => {
+            expect(body.msg).toBe("path not found");
+          });
+      });
+    })
```

Comment:
This test doesn't belong in this describe block, as it relates to all bad paths, not just users